src/app/components/header/header.component.html

```diff
@@ -1,7 +1,8 @@
 <ion-header>
+  <ion-title>My Day</ion-title>
```

Comment:
Why common header have title as My Day?

Reply:
Its is not a common header, It is a local header of "My Day" page

Reply:
It is used only in my day as of today that does not mean that it is my day header. This should be common header for app. So keep it generic.